packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -122,7 +122,9 @@ export default class SSRRenderer extends BaseRenderer {
     }
 
     // Prepend scripts
-    APP += this.renderScripts(renderContext)
+    if (this.options.render.bundleRenderer.inject) {
```

Comment:
Shouldn't we provide a default value for `inject` (`true`) to avoid a breaking change? :thinking:

Reply:
@manniL Nice catch. Yes should be added here:

https://github.com/nuxt/nuxt.js/blob/dev/packages/config/src/config/render.js#L4

Reply:
I think it should be `this.options.render.bundleRenderer.inject !== false`

Reply:
Even better :+1:

Reply:
sorry, it's my first time to pull request. so what should I do now？

Reply:
I have a new commit and I have pushed to the 'dev' branch.